src/pwm.rs

```diff
@@ -2,14 +2,14 @@
   # Pulse width modulation
 
   The general purpose timers (`TIM2`, `TIM3`, and `TIM4`) can be used to output
-  pulse width modulated signals on some pins. The timers support up to 4 simultaneous
-  pwm outputs in separate `Channels`
+  pulse width modulated signals on some pins. The timers support up to 4
+  simultaneous pwm outputs in separate `Channels`
 
-  ## Usage
+  ## Usage for pre-defined channel combinations
 
-  Start by setting all the pins for the timer you want to use to alternate
-  push pull pins. The `Pins` trait shows the mapping between timers, output pins
-  and channels.
+  Crate defines only basic channel combinations for default AFIO remappings,
```

Comment:
I think this should say "This crate only defines..."

Reply:
Done